AppleALC/ALCUserClientProvider/ALCUserClientProvider.cpp

```diff
@@ -0,0 +1,81 @@
+//
+//  ALCUserClientProvider.cpp
+//  AppleALC
+//
+//  Created by Nick on 10/14/20.
+//  Copyright © 2020 vit9696. All rights reserved.
+//
+
+#include "ALCUserClientProvider.hpp"
+
+OSDefineMetaClassAndStructors(ALCUserClientProvider, IOService);
+
+bool ALCUserClientProvider::start(IOService* provider) {
+	if (!super::start(provider))
+		return false;
+	
+	mHDACodecDevice = IOService::waitForMatchingService(IOService::nameMatching(kIOHDACodecDevice), 3000);	// Wait for 3s
+	
+	if (!mHDACodecDevice)
+	{
+		DBGLOG("client", "HDACodecDevice not found");
```

Comment:
Is this normal, so we put it to a debug log?

Reply:
Actually it should be a SYSLOG. Fixed.